src/HotChocolate/Mutable/src/Types.Mutable/BuiltIns/BuiltIns.cs

```diff
@@ -140,7 +160,6 @@ public static bool IsBuiltInDirective(string name)
             Skip.Name => true,
             Deprecated.Name => true,
             SpecifiedBy.Name => true,
-            OneOf.Name => true,
```

Comment:
TODO: Should `@oneOf` still be serialized into the schema or does it behave like the other spec directives now?